dev-python/plotly/plotly-4.10.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Browser-based graphing library for Python"
+HOMEPAGE="https://plot.ly/python/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/retrying[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+
+python_install_all() {
```

Comment:
any tests here? all things equal, tests are more important in python packages than C/C++ packages

Reply:
for sake of others who might read.
these packages don't contain tests, they are made from a single source tarball which contains 6 different packages (broken up later for pypi releases), so we can't use github releases as well.